frigate/config.py

```diff
@@ -349,6 +349,9 @@ class BirdseyeModeEnum(str, Enum):
 class BirdseyeConfig(FrigateBaseModel):
     enabled: bool = Field(default=True, title="Enable birdseye view.")
     restream: bool = Field(default=False, title="Restream birdseye via RTSP.")
+    sort_cameras: bool = Field(
+        default=False, title="Sort cameras by position and name."
+    )
```

Comment:
Do we need an option for this? If all start with an equal position then I think alphabetical sorting wouldn't be a problem as the default.

Reply:
I'm not sure if we need an option for this, in a way I was trying to preserve the existing default behaviour in case this causes any issues.
I don't feel too strongly about this, happy to change the default behaviour.